mersenne-31/src/extension2.rs

```diff
@@ -0,0 +1,42 @@
+use core::ops::Mul;
+
+use p3_field::{define_ext, impl_ext_af, impl_ext_f, AbstractField, Field};
+
+use crate::Mersenne31;
+
+define_ext!(pub CM31([Mersenne31; 2]));
```

Comment:
I know it's obvious in this case but maybe add a comment defining what this field is.